src/components/grid-aware/StatsBarBlock/StatsBarBlock.module.css

```diff
@@ -0,0 +1,218 @@
+.gray {
+  background-color: var(--color-gray-300);
+}
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+.bleedBackground {
+  grid-column: 2 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0 150px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 38px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column: 2 / span 10;
+  grid-row: 3;
+  margin-top: 54px;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  grid-column: 2 / span 8;
+  margin: 0;
+}
+
+.blue {
+  color: var(--color-sheltertech-blue);
+}
+
+.subtitle {
+  font: var(--font-body-medium);
+  grid-column: 2 / span 8;
+  margin-top: 20px;
+}
```

Comment:
I was looking carefully at the spacing in the developer console, and it looks like `h2` elements have a default `margin-bottom` of about 16px, which you may want to explicitly set to 0 to exactly match the design.